Jenkinsfile

```diff
@@ -24,7 +24,7 @@ pipeline {
             steps {
                ansiColor('xterm') {
                   sh 'git fetch --tags'
-                  sh './build.sh cleanup'
+                  sh './build.sh cleanup vicoconsulting'
```

Comment:
Can we gave that more generic? :-)

Reply:
feel free, i did not check every change as the UI display issue is annoying. removing branding is fine for me if command line args can be used

Reply:
Can we remove the company name?

Reply:
updated